### PR TITLE
[ready-for-core-team-review]--CRM-16189, generalized code, removed hardcoded values, used function

### PR DESCRIPTION
**Pull request by pradpnayak**

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
test this please

File: CRM/Financial/Form/FinancialTypeAccount.php [hunk 1]
```diff
@@ -141,12 +141,13 @@ public function buildQuickForm() {
...
     }
-    $AccountTypeRelationship = CRM_Core_PseudoConstant::get('CRM_Financial_DAO_EntityFinancialAccount', 'account_relationship');
+    $params['orderColumn'] = 'label';
+    $AccountTypeRelationship = CRM_Core_PseudoConstant::get('CRM_Financial_DAO_EntityFinancialAccount', 'account_relationship', $params); 
     if (!empty($AccountTypeRelationship)) {
       $element = $this->add('select',
         'account_relationship',
         ts('Financial Account Relationship'),
-        array('select' => '- select -') + $AccountTypeRelationship,
+        array('select' => '- Select Financial Account Relationship -') + $AccountTypeRelationship,
```
Comment (eileenmcnaughton) [inline CRM/Financial/Form/FinancialTypeAccount.php]: ts ?

**Comment by eileenmcnaughton** [comment]
looks like an improvement

**Comment by pradpnayak** [comment]
jenkins, retest this please

**Pull request merged**